app/Http/Controllers/Store/Auth/LoginController.php

```diff
@@ -58,12 +58,12 @@ public function login(Request $request)
             'password' => 'required|min:6',
         ]);
         // Attempt to log the user in
-        if (Auth::guard('web')->attempt(
+        if (Auth::guard('store')->attempt(
```

Comment:
Nice!

Reply:
yeah. I missed that. I was being very dumb. for 2 days!